db/dcrpg/go.mod

```diff
@@ -2,6 +2,10 @@ module github.com/decred/dcrdata/db/dcrpg/v3
 
 go 1.11
 
+replace github.com/decred/dcrdata/testutil/dbconfig => ../../testutil/dbconfig
```

Comment:
So the db config can be changed and modified tests run without banging your head against the wall.